packages/client-sdk-nodejs/src/internal/grpc/retry-interceptor.ts

```diff
@@ -91,10 +96,14 @@ export class RetryInterceptor {
                       savedMessageNext(savedReceiveMessage);
                       next(status);
                     } else {
+                      const whenToRetryWithJitter = addJitter(whenToRetry);
```

Comment:
Possibly. I just added jitter here because there was a [TODO note in this file](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/client-sdk-nodejs/src/internal/grpc/retry-interceptor.ts#L43) about it

Reply:
agreed with michael, the jitter should go inside the retry strategy, and we can just remove that old comment. Thanks!